jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -299,10 +299,10 @@ String getTargetImage() {
    */
   Set<String> getTargetImageAdditionalTags() {
     String property = getProperty(PropertyNames.TO_TAGS);
-    if (property != null) {
-      return new HashSet<>(ConfigurationPropertyValidator.parseListProperty(property));
-    }
-    return new HashSet<>(to.tags);
+    List<String> tags =
+        property != null ? ConfigurationPropertyValidator.parseListProperty(property) : to.tags;
+    tags.forEach(tag -> Preconditions.checkNotNull(tag, "null or empty tag"));
```

Comment:
I'd like to have the user see a more friendly and useful message. This is the Maven plugin code, so I guess it's OK to have a very Maven-specific message here. Or should we use `HelpfulSuggestions`?